db/seeds/dev/residents.js

```diff
@@ -20,6 +20,108 @@ exports.seed = function(knex) {
         living_preferences: "",
         semester: "Spring 2021",
         additional_notes: "I need help feeding my cat, Mittens. Please don't eat peanuts around me, I am terribly allergic."
+      },
+      { 
+        name: "Ralph",
+        photo: "https://previews.123rf.com/images/natakot/natakot1810/natakot181001281/110094284-happy-old-man-showing-thumbs-up-working-out-with-a-dumbbell.jpg",
+        age: "75",
+        height: "5'11\"",
+        weight: "190 lbs",
+        pcp: "Dr. Segura",
+        ambulatory: "yes",
+        help_with_medications: "no",
+        other_impairments: "early stage memory loss",
+        previous_career: "Small business owner",
+        interests: "Weightlifting, billiards, NY stock exchange",
+        pets: "none",
+        living_preferences: "I like to wake up early and stay active.",
+        semester: "Spring 2021",
+        additional_notes: "Once a month, my 12-year-old grandson comes to stay with me. He's a good kid and is fine sleeping on the couch."
+      },
+      { 
+        name: "Wendy",
+        photo: "",
```

Comment:
We need to add values to some of these fields, and maybe create just a few more instances